src/components/Pagination.vue

```diff
@@ -40,8 +40,8 @@ export default class Pagination extends Vue {
 
   @VQBModule.Mutation setCurrentPage!: MutationCallbacks['setCurrentPage'];
 
-  get showPager() {
-    return this.pageRows.max !== this.totalCount;
+  get showPager(): boolean {
```

Comment:
Excellent, thanks! Could you please add a test case to expose this bug (and make sure it's fixed :)) ?

Reply:
@adimasci , i thought these test was already enough (already implemented in `pagination-component.spec.ts`):
```
  it('should hide the pagination navigation if there is only one page', () => {
    const store = setupMockStore({
      dataset: {
        headers: [{ name: 'city' }, { name: 'population' }, { name: 'isCapitalCity' }],
        data: [
          ['Paris', 10000000, true],
          ['Marseille', 3000000, false],
        ],
        paginationContext: {
          totalCount: 2,
          pageno: 1,
          pagesize: 2,
        },
      },
      pagesize: 2,
    });
    const wrapper = mount(Pagination, { localVue, store });
    expect(wrapper.find('.pagination__list').exists()).toBeFalsy();
    expect(wrapper.find('.pagination-counter__current-min').exists()).toBeFalsy();
    expect(wrapper.find('.pagination-counter__current-max').exists()).toBeFalsy();
    expect(wrapper.find('.pagination-counter__total-count').text()).toEqual('2 rows');
  });
```

Let me know, if it doesn't respond to your issue :smile:

Reply:
If needed i also can add a specific test (going to last page, to check pagination on last page only) but isn't it too specific ?

Reply:
Well… as far as I understand it, there was a bug and yet, tests were passing, therefore I think there is a hole to fill :) Or am I missing something?